QEfficient/utils/_utils.py

```diff
@@ -186,3 +187,17 @@ def padding_check_and_fix(tokenizer: Union[PreTrainedTokenizer, PreTrainedTokeni
             tokenizer.pad_token_id = tokenizer.eos_token_id
         else:
             tokenizer.pad_token_id = tokenizer.vocab_size - 1
+
+
+def generate_sha256_hash(data: str) -> str:
+    """
+    Generate a SHA-256 hash for the given data.
+
+    :param data: The input data to hash.
```

Comment:
can you add more details on what this data excatly is? `model_card` name?

Reply:
Any string data can be converted to hash. I added it as a generic method under utils.